app/views/home.scala.html

```diff
@@ -11,7 +11,7 @@
 @import ore.project.Categories._
 @import ore.project.ProjectSortingStrategies.{values, _}
 @import db.impl.UserBase
-@(models: Seq[Project], visibleCategories: Option[Array[Category]], query: Option[String], page: Int,
+@(models: Seq[Project], total: Int, visibleCategories: Option[Array[Category]], query: Option[String], page: Int,
```

Comment:
Perhaps name this something other than just `total`? It's unclear what it is without seeing where it's used.